src/AppInstallerRepositoryCore/Rest/Schema/Json/ManifestDeserializer.cpp

```diff
@@ -0,0 +1,397 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "Rest/Schema/1_0/Interface.h"
+#include "Rest/Schema/IRestClient.h"
+#include "Rest/HttpClientHelper.h"
+#include "cpprest/http_client.h"
+#include "cpprest/json.h"
+#include "ManifestDeserializer.h"
+#include "JsonHelper.h"
+
+using namespace AppInstaller::Manifest;
+
+namespace AppInstaller::Repository::Rest::Schema::Json
+{
+    namespace
+    {
+        // Manifest response constants specific to this deserializer
+        constexpr std::string_view PackageIdentifier = "PackageIdentifier"sv;

```

Comment:
Proof that having the tree walking interface and single definition of the manifest parser would be much less code.  This would literally not need to know anything about the manifest structure (assuming that we actually defined the same tree for the REST response... please tell me we did that).

Reply:
Not heavily reviewed because I think we should do that work sooner rather than later.